src/test/lib/ApiTest.cpp

```diff
@@ -4227,6 +4227,108 @@ void QuicTest_QUIC_PARAM_CONN_STATISTICS_V2_PLAT(MsQuicRegistration& Registratio
     }
 }
 
+
+void QuicTest_QUIC_PARAM_CONN_ORIG_DEST_CID(MsQuicRegistration& Registration, MsQuicConfiguration& ClientConfiguration) {
+    //
+    // This is the unit test for checking to see if a server has the correct original dest CID.
+    //
+    TestScopeLogger LogScope0("QUIC_PARAM_CONN_ORIG_DEST_CID");
+    {
+        MsQuicConnection Connection(Registration);
+        TEST_QUIC_SUCCEEDED(Connection.GetInitStatus());
+        TEST_QUIC_SUCCEEDED(
+        MsQuic->ConnectionStart(
+            Connection.Handle,
+            ClientConfiguration,
+            QUIC_ADDRESS_FAMILY_INET,
+            "localhost",
+            4433));
```

Comment:
```suggestion
        TEST_QUIC_SUCCEEDED(
          Connection.Start(
              ClientConfiguration,
              QUIC_ADDRESS_FAMILY_INET,
              "localhost",
              4433));
```
Minor formatting fix and uses the helper function.

Reply:
will update in next push

Reply:
done